runtime/src/accounts_db.rs

```diff
@@ -8990,7 +8990,7 @@ impl AccountsDb {
                 if not_sparse || too_small_to_shrink {
                     return 0;
                 }
-                info!(
+                trace!(
```

Comment:
the rest of the logs in this function are `trace`, so that's what i went with.  `debug`'s probably fine to